src/twisted/test/proto_helpers.py

```diff
@@ -10,7 +10,22 @@
 from twisted.internet import testing
 
 
-__all__ = testing.__all__
+__all__ = [
```

Comment:
This looks good and I think it's the right thing to do.

For this PR, I think that it would help to add a comment that this is a snapshot of public API and we plan to deprecate this module... maybe add the link to the ticket asking for the deprecation.

Assuming that we agree do deprecate this module. 

The comment should also mention that no other member should be added to `__all__`